cmd/commands/root.go

```diff
@@ -102,12 +102,12 @@ func GetClientsets(ctx context.Context) *k8sutil.Clientsets {
 		logging.Fatal(err)
 	}
 
-	PreValidationCheck(ctx, clientsets, OperatorNamespace, CephClusterNamespace)
+	PreValidationCheck(ctx, clientsets, OperatorNamespace, CephClusterNamespace, checkRookVersion)
 
 	return clientsets
 }
 
-func PreValidationCheck(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace string) {
+func PreValidationCheck(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace string, checkRookVersion bool) {
```

Comment:
Is this parameter really about whether we require the operator pod to be running? While we implement that check by checking the rook version, I just want to be clear when `false` or `true` should be passed to this method.
```suggestion
func PreValidationCheck(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace string, requireOperatorPodRunning bool) {
```

Reply:
removed the check